modules/dtg/manifests/backup.pp

```diff
@@ -76,20 +76,22 @@
 # The name of the backup will be used as the directory name for the subdir containing the backups
 # user is the user to ssh in as
 # host is the host to ssh into
-define dtg::backup::hostsetup($user, $host, $weekday) {
+define dtg::backup::hostsetup($user, $group = $dtg::backup::host::user, $host, $weekday) { #needs to have a configurable backup group + default
```

Comment:
Does this work? I would have thought that the optional parameter needs to be the last parameter?